doc/tor.md

```diff
@@ -23,6 +23,9 @@ type, including Tor v2 and v3. This is useful to see how many onion addresses
 are known to your node for `-onlynet=onion` and how many Tor v3 addresses it
 knows when upgrading to current and future Tor releases that support Tor v3 only.
 
+To fetch a number of onion addresses that your node knows, for example seven
+addresses, use the `getnodeaddresses 7 onion` RPC.
```

Comment:
nit: we can't do this yet it seems

Reply:
We can since https://github.com/dashpay/dash/pull/5978 (https://github.com/bitcoin/bitcoin/pull/21594)

https://github.com/dashpay/dash/blob/00e556ac7aaa19db98bcff9473264686365bd2f7/src/rpc/net.cpp#L908-L912

Reply:
Nevermind; this does work!